system/core/base.php

```diff
@@ -340,7 +306,8 @@ function hypha_getCss() {
 		$contents - css file contents
 	*/
 	function hypha_setCss($contents) {
-		Hypha::$data->css->writeWithLock($contents);
+		global $O_O;
+		$O_O->data->css->writeWithLock($contents);
```

Comment:
This should be `themeCss` in stead of `css`; `$O_O->data->css->writeWithLock($contents);`

Reply:
Fixed.